src/main/java/uk/gov/hmcts/reform/prl/enums/manageorders/SelectTypeOfOrderEnum.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.hmcts.reform.prl.enums.manageorders;
+
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonValue;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import lombok.RequiredArgsConstructor;
+import uk.gov.hmcts.reform.prl.enums.CustomEnumSerializer;
+
+@RequiredArgsConstructor
+@JsonSerialize(using = CustomEnumSerializer.class)
+public enum SelectTypeOfOrderEnum {
+    @JsonProperty("interim")
+    interim("interim", "Interim"),
+    @JsonProperty("general")
+    general("general", "General"),
+    @JsonProperty("finl")
+    finl("finl", "Final");
```

Comment:
can we rename it to final from finl please.

Reply:
there are multiple places in ccd-def it used similar way so changing it will require change in there too.